include/WCSimRootEvent.hh

```diff
@@ -165,6 +165,26 @@ public:
   ClassDef(WCSimRootCherenkovHitTime,2)
 };
 
+// Scattering and reflection history for each Cherenkov hit (photon)
+class WCSimRootCherenkovHitHistory : public TObject {
+
+private:
+  
+  Int_t   fScat;
+  std::vector<Int_t> fReflec;
+
+public:
+  WCSimRootCherenkovHitHistory() {}
+  WCSimRootCherenkovHitHistory(Int_t scat, std::vector<Int_t> refle);
+  virtual ~WCSimRootCherenkovHitHistory() { }
+  bool CompareAllVariables(const WCSimRootCherenkovHitHistory * c) const;
+
+  Int_t     GetScatter() const { return fScat; }
+  std::vector<Int_t> GetReflection() const { return fReflec; }
```

Comment:
Could call it `GetReflectionSurfaces()`
(at the very least, it should be plural)